src/Pyro4/core.py

```diff
@@ -221,7 +221,7 @@ class Proxy(object):
         ["__getnewargs__", "__getnewargs_ex__", "__getinitargs__", "_pyroConnection", "_pyroUri",
          "_pyroOneway", "_pyroMethods", "_pyroAttrs", "_pyroTimeout", "_pyroSeq", "_pyroHmacKey",
          "_pyroRawWireResponse", "_pyroHandshake", "_pyroMaxRetries", "_pyroSerializer", "_Proxy__async",
-         "_Proxy__pyroHmacKey", "_Proxy__pyroTimeout", "_Proxy__pyroConnLock"])
+         "_Proxy__pyroHmacKey", "_Proxy__pyroTimeout", "_Proxy__pyroConnLock", "__transient__"])

```

Comment:
Sure. Once the new `cloudpickle` version is release, I will revert this as well to check Travis is happy with everything too. Might do it in new commits to make review easier for you, then I can squash everything in a single commit (don't think it makes sense to split this in more than 1 commit).